### PR TITLE
peribolos: Normalize GitHub usernames when listing invites

### PR DESCRIPTION
**Pull request by cblecker**
/assign @fejta @BenTheElder 
FYI: @kubernetes/owners

### CODE REVIEW

File: prow/cmd/peribolos/main.go [hunk 1]
```diff
@@ -692,7 +692,7 @@ func orgInvitations(opt options, client inviteClient, orgName string) (sets.Stri
 		if i.Login == "" {
 			continue
 		}
-		invitees.Insert(i.Login)
+		invitees.Insert(github.NormLogin(i.Login))
```
Comment (BenTheElder) [inline prow/cmd/peribolos/main.go]: lol I am the best test case I assume /s 👍

Comment (cblecker) [reply prow/cmd/peribolos/main.go]: Thanks for breaking everything with your silly capitals 🤣

**Comment by cblecker** [comment]
Discovered [here](https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/post-org-peribolos/5/?log#log):
```
{"component":"peribolos","level":"info","msg":"Set rajakavitha1 as a member of kubernetes-incubator","time":"2018-09-19T00:59:35Z"}
```
The above member already had an invite to the org.

---
Verified the new test fails without the change to main.go:
```
--- FAIL: TestOrgInvitations (0.00s)
    --- FAIL: TestOrgInvitations/ensure_case_normalization (0.00s)
        main_test.go:1952: sets.String{"MiXeD":sets.Empty{}, "lower":sets.Empty{}, "UPPER":sets.Empty{}} != expected sets.String{"mixed":sets.Empty{}, "lower":sets.Empty{}, "upper":sets.Empty{}}
```

**Pull request merged**